README.md

```diff
@@ -16,16 +16,27 @@ As an upstream (dangling-backend) [long-api](https://datmt.com/backend/docker-im
 
 
 ## Building base environment
+### Default: use only single core to make (compile) process:
 
 ```
 docker-compose up -d --build
 ```
+### Configure number of threads for make (compile) process. Use 4 CPU cores example:
+```
+docker-compose build --build-arg MAKE_THREADS=4 && docker-compose up -d
```

Comment:
Shouldn't it be defined explicitly in `docker-compose.yaml`?
```
  base:
    image: nginx-with-modsec-base
    build:
      dockerfile: ./docker/nginx-with-modsec-base/Dockerfile
      args:
        - MAKE_THREADS=1
```

Reply:
I added `MAKE_THREADS` arg to docker-compose files. 
But it is more handy to single `--build-arg MAKE_THREADS=4` when running `docker compose build` instead of changing `MAKE_THREADS` arg in multiple places in docker-compose files.
I also switched from `docker-compose` to `docker compose` command, more info about differences here:
https://docs.docker.com/compose/migrate/